website/static/js/registrationUtils.js

```diff
@@ -981,7 +983,7 @@ RegistrationEditor.prototype.context = function(data, $root, preview) {
     });
 
     if (this.extensions[data.type]) {
-        return new this.extensions[data.type](data, $root, preview);
+        return new this.extensions[data.type](data, $root.pk, preview);
```

Comment:
The root wasn't being used by either of the functions this called. I needed to get the id to trickle down to the lower reaches of the gargantuan pile, passing just root caused errors, so I passed only part of it. https://github.com/CenterForOpenScience/osf.io/pull/5907/files#r71750654